modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1311,6 +1323,84 @@ public void testBindObjectDefinitions() throws Exception {
 			new String[] {"C_A", "C_AA", "C_AAA", "C_AAB"});
 	}
 
+	@Test
+	public void testDeleteCompanyObjectDefinitions() throws Exception {
+		String originalName = PrincipalThreadLocal.getName();
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		Company company = CompanyTestUtil.addCompany();
+
+		PortalInstances.initCompany(company);
+
+		try (SafeCloseable safeCloseable =
+				CompanyThreadLocal.setWithSafeCloseable(
+					company.getCompanyId())) {
+
+			User user = UserTestUtil.getAdminUser(company.getCompanyId());
+
+			Assert.assertNotNull(user);
```

Comment:
@seltong do we really need this assert?

Reply:
I believe we don't need this, so I removed